spec/controllers/content_items_controller_spec.rb

```diff
@@ -41,6 +41,16 @@
 
       expect(subject).to render_template(:index)
     end
+
+    it "assigns the lists of taxonomies and organisations ordered ASC by title" do
+      organisations = [{ title: "ABC" }, { title: "BCD" }].map { |o| create(:organisation, o) }
+      taxons = [{ title: "ABC" }, { title: "BCD" }].map { |t| create(:taxonomy, t) }
```

Comment:
Made test explicit in testing number or orgs/taxons PLUS that they were ordered alphabetically.